go.mod

```diff
@@ -200,3 +200,5 @@ require (
 )
 
 replace github.com/robfig/cron/v3 => github.com/unionai/cron/v3 v3.0.2-0.20210825070134-bfc34418fe84
+
+replace github.com/flyteorg/flyteidl => github.com/flyteorg/flyteidl v1.1.2-0.20220523141814-4fbb9bb1c89e
```

Comment:
friendly reminder to not merge this until after the idl version is updated :)

Reply:
updated it